assets/js/weatherApp.js

```diff
@@ -0,0 +1,44 @@
+let weather = {
+    "apikey": "375090722ff9b13931a9ed44d6f32f14",
+
+    // Fetch data from openweathermap API call.
+    fetchWeather: function (city) {
+        fetch(
+            "http://api.openweathermap.org/data/2.5/weather?q=" 
+            + city 
```

Comment:
i was use openweathermap forecast in several projects and i recommend use `lat & lon` coordinates prefer city name. If you use city name this may not work well for people from other countries or small towns because of problems with synonyms.

See `By geographic coordinates` for example https://openweathermap.org/current

https://openweathermap.org/current